Dockerfile

```
@@ -3,27 +3,36 @@ ARG AIRFLOW_VERSION=2.9.3
 FROM apache/airflow:${AIRFLOW_VERSION}-python${PYTHON_VERSION}
 
-# ======== system packages (optional) ========
+# ======== system packages ========
 USER root
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-      tzdata ca-certificates curl unzip wget \
-      build-essential \
+        tzdata ca-certificates curl unzip wget build-essential \
     && rm -rf /var/lib/apt/lists/*
+<<<<<<< HEAD
```

Hadolint (2.14.0)

[error] unexpected '<'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)